src/components/IdScanner/IdScanner.tsx

```diff
@@ -148,7 +148,7 @@ export const IdScanner: FunctionComponent<IdScanner> = ({
 
   useEffect(() => {
     const askForCameraPermission = async (): Promise<void> => {
-      const { status } = await Permissions.askAsync(Permissions.CAMERA);
+      const { status } = await Camera.requestPermissionsAsync();
```

Comment:
Can we use `granted` instead? (see [here](https://docs.expo.io/versions/v42.0.0/sdk/permissions/#permissionresponsegranted))

It seems more appropriate for determining if the user has granted sufficient permissions to access the camera.

Reply:
Never mind, seems like the [documentation](https://docs.expo.io/versions/latest/sdk/camera/) uses `status`, let's go with that!